prisma/datamodel.graphql

```diff
@@ -10,6 +10,8 @@ type Profile {
   titleFr: String
   team: Team @relation(name:"TeamMembers")
   ownerOfTeams: [Team!]! @relation(name:"OwnerOfTeam")
+  outstandingApprovals: [Approval] @relation(name:"Approver")
```

Comment:
Needed to add the 2 fields for approver and submitter on the Profile schema because of the `@relation` field defined on the Approval schema